addOns/commonlib/CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - A generic UI component for keeping menu items sorted.
 
 ### Changed
-- Add solution to 'Brute Forcing Credit Card Information', 'Content Spoofing', 'Credential and Session Prediction', 'XML Injection' and 'XML External Entities' vulnerabilities (Issue 8056).
+- Add solution to 'Fingerprinting', 'Brute Forcing Credit Card Information', 'Content Spoofing', 'Credential and Session Prediction', 'XML Injection' and 'XML External Entities' vulnerabilities (Issue 8056).
```

Comment:
A new single entry under Unreleased

Reply:
when I click compare on the force push, I see so many changes like versions, Have I done anything wrong?

Reply:
Seems fine to me. The changelog just needs to be properly updated.

Reply:
![image](https://github.com/zaproxy/zap-extensions/assets/63240815/c9a9e3ce-6462-4ef5-87cf-c0fc98747102)
Any suggestion ?

Reply:
Yes the addon was released this morning so the only “new” change is this one. Move it to the unreleased section add a new changed heading etc

Amend your commit and force push again.